public/js/download.js

```diff
@@ -87,7 +87,7 @@ function testDownload(fileSize) {
     let start = new Date().getTime();
     $.ajax({
         type: "GET",
-        url: "/download/testfile.bin?id=" + start,
+        "url": Routing.generate("pelagos_homepage") + "/images/testfile.bin?id=" + start,
```

Comment:
Just use asset('path to file')

Reply:
Ok, can't use asset tag in Javascript, so you'll have to set it somewhere in the template.

Reply:
https://symfony.com/doc/4.3/frontend/encore/server-data.html

Reply:
Ok, forget all that, I think it's easier if you just want to the base path, just get it from Routing like this: `Routing.getBaseUrl()`